Sources/Tokenizers/PreTokenizer.swift

```diff
@@ -53,6 +54,7 @@ struct PreTokenizerFactory {
         case .Punctuation: return PunctuationPreTokenizer(config: config)
         case .Digits: return DigitsPreTokenizer(config: config)
         case .Split: return SplitPreTokenizer(config: config)
+        case .Whitespace: return WhitespacePreTokenizer(config: config)
         default       : fatalError("Unsupported PreTokenizer type: \(typeName)")
```

Comment:
```suggestion
        case .Whitespace: return WhitespacePreTokenizer(config: config)
        default: fatalError("Unsupported PreTokenizer type: \(typeName)")
```

Can we take the opportunity to remove this old alignment spacing that is no longer up to date? 🙏

Reply:
done